zebra-consensus/src/block/check.rs

```diff
@@ -71,6 +71,12 @@ pub fn difficulty_is_valid(
         ))?;
     }
 
+    // # Consensus
+    //
+    // > The block MUST pass the difficulty filter.
+    //
+    // https://zips.z.cash/protocol/protocol.pdf#blockheader
+    //
     // The difficulty filter is also context-free.
     //
     // ZIP 205 and ZIP 208 incorrectly describe testnet minimum difficulty blocks
```

Comment:
This was fixed in https://github.com/zcash/zips/pull/417 .

Reply:
Thank you @daira, i removed that comment in https://github.com/ZcashFoundation/zebra/pull/3296/commits/65d9a79dc93bf7b8711ee5b2935abfc34f3d6020